src/config/settings/base.py

```diff
@@ -200,7 +200,7 @@
         "BACKEND": "django.template.backends.django.DjangoTemplates",
         "APP_DIRS": True,
         "DIRS": [
-            Path(PROJECT_ROOT_DIR) / "dw-design-system",
+            Path(PROJECT_ROOT_DIR) / "src/dw_design_system",
```

Comment:
```suggestion
            Path(PROJECT_ROOT_DIR) / "src" / "dw_design_system",
```
we definitely need the "src" do we?

Reply:
yes, doesn't load the templates without src @marcelkornblum